src/test/java/com/xiao/users/service/impl/UserServiceImplTest.java

```diff
@@ -84,4 +89,25 @@ void testFindUserById_whenNotFound() {
             assertEquals(ex.getMessage(), String.format("%s not found with the given input data %s : '%s'", "User", "id", userId));
         }
     }
+
+    @Test
+    void testFindAllUser(){
+
+        List<User> userList = Arrays.asList(UserUtil.buildUser(), UserUtil.buildUser());
+        Page<User> expectedPage = new PageImpl<>(userList, PageRequest.of(0, 10), userList.size());
+        when(userRepository.findAll(PageRequest.of(0,10))).thenReturn(expectedPage);
+
+        Page<UserDto> actualPage = userService.findAllUser(0, 10);
+        Page<UserDto> expectedResult = expectedPage.map(userMapper::userToUserDto);
+
+        assertEquals(expectedResult.getContent(), actualPage.getContent());
```

Comment:
Asserting full content objects like this can potentially lead to issues. 
You don't need to fix it now, but let's wait and see that you may encounter trouble with the code in the future.